### DISCUSSION TITLE
HH damping calibration using damping experimental data

### DISCUSSION THREAD

**Discussion by domniki** [Caltech-geoquake/PySeismoSoil#17] [Q&A] [answered]
instead of using Darendeli curves from Vs-profile input

**Comment by jsh9** [comment, marked as answer]
We can readily do this:

- First, we define a `Multiple_Damping_Curves` object with custom curves (instead of from Darendeli's formulas)
- Then, we use the `get_all_HH_x_params()` method to do non parametric curve fitting (based on the evolutionary algorithm)

Here is an example: https://github.com/jsh9/PySeismoSoil/blob/455be0fb5c46ab48cbe107b8ec91f852e732d28a/examples/Demo_04_Curves.ipynb (see Section 3: Multiple_Damping_Curves class)
